frontend/app/components/admin/user/table-row.js

```diff
@@ -24,14 +24,9 @@ export default class AdminUserTableRowComponent extends Component {
   }
 
   get isRoleEditingDisabled() {
-    if (this.isOwnUserOrRoot())
-      return true;
+    if (this.isOwnUserOrRoot()) return true;
 
-    if (this.userService.isAdmin || this.isConfAdminChangingAdmin()) {
-      return false;
-    } else {
-      return true;
-    }
+    return !(this.userService.isAdmin || this.isConfAdminChangingAdmin());
```

Comment:
Just asked myself who wrote that horrible code.... remembered it was me on monday.... thanks for cleaning this up and let's pretend this has never existed in the first place